NEMP/mods.json

```diff
@@ -3723,15 +3714,159 @@
             "name": "cooking-for-blockheads"
         }
     },
+    "AdobeBlocks": {
+        "function": "CheckCurse",
+        "active": true,
+        "curse": {
+            "regex": "AdobeBlocks(?:-Forge)?(?:-(?P<mc>.+?))?-(?P<version>[0-9]+(?:\.[0-9]+)+)",
```

Comment:
For some reason I'm struggling with GitHub PR reviews today... These changes are still a concern.